examples/kserve_deployment/README.md

```diff
@@ -0,0 +1,427 @@
+the PyTorch model, solver and penalty for the Scikit-learn logical regression) 
+can be supplied as command-line arguments to the `run.py` Python script. 
+
+The example contains three pipelines:
+    * `kserve_sklearn_pipeline`: trains a classifier using Scikit-learn and deploys it to KServe with SKLearn MLServer Runtime Server.
+    * `kserve_pytorch_pipeline`: trains a classifier using PyTorch and deploys it to KServe with TorchServe Runtime Server.
+    * `inference_pipeline`: runs predictions on the served models.
+
+Running the pipelines to train the classifiers and then deploying them to 
+KServe requires preparing them into an exact format that is expected 
+by the runtime server, storing them into remote storage or a persistent volume 
+in the cluster and giving the path to KServe as the model uri with the right permissions. 
+By default, ZenML's KServe integration will try to cover that for you 
+by automatically loading, preparing and then saving files to the Artifact Store 
+active in the ZenML stack. However, for some frameworks (e.g. Pytorch) you will still need 
+to provide some additional files that Runtime Server needs to be able to run the model. 
+
+The KServe deployment server is provisioned remotely as a Kubernetes
+resource that continues to run after the deployment pipeline run is complete.
+Subsequent runs of the deployment pipeline will reuse the existing deployment
+server and merely update it to serve the more recent model version.
+
+The deployment pipeline has caching enabled to avoid re-training and
+re-deploying the model if the training data and hyperparameter values don't
+change. When a new model is trained that passes the accuracy threshold
+validation, the pipeline automatically updates the currently running KServe
+deployment server so that the new model is being served instead of the old one.
+
+The inference pipeline load image from the local filesystem and perform 
+online predictions on the running KServe inference service.
+
+
+# 🖥 Local Stack
+
+### 📄 Prerequisites 
+
+For the ZenML KServe deployer to work, these things are required:
+1. Access to a running [Kubernetes cluster](https://kubernetes.io/docs/tutorials/cluster-administration/) 
+    The example accepts a `--kubernetes-context` command-line argument. This Kubernetes context needs 
+    to point to the Kubernetes cluster where KServe model servers will be deployed. If the context 
+    is not explicitly supplied to the example, it defaults to using the locally active context.
+
+2. KServe must be installed and running on the Kubernetes cluster. (more information about how to
+    install KServe can be found below or in the [KServe documentation](https://kserve.github.io/website/)).
+
+3. KServe must be able to access whatever storage is used by the ZenML to save the artifact, Since the
+    KServe is installed in the Kubernetes cluster, a local filesystem storage can't be used.
```

Comment:
```suggestion
3. KServe must be able to access whatever storage is used by ZenML to save the artifact. Since 
    KServe is installed in the Kubernetes cluster, a local filesystem storage can't be used.
```